cypress/integration/integration_spec.js

```diff
@@ -558,11 +560,10 @@ it('should submit full awards for all application', () => {
                 };
                 break;
             case 'Scotland':
-                location = sample([
```

Comment:
We had this mostly when we were submitting lots of random test data to Salesforce, we have easier ways of doing this now (and less of a requirement) so we can reduce the randomness here.